scripts/demo-email-notifications/index.js

```diff
@@ -1,13 +1,16 @@
 const Kafka = require('no-kafka')
 const fs = require('fs')
 const config = require('config')
+const axios = require('axios')
+const _ = require('lodash')
 const moment = require('moment')
 const handlebars = require('handlebars')
 const logger = require('../../src/common/logger')
 const { Interview, JobCandidate, ResourceBooking } = require('../../src/models')
 const { Interviews } = require('../../app-constants')
 
 const consumer = new Kafka.GroupConsumer({ connectionString: process.env.KAFKA_URL, groupId: 'test-render-email' })
+const slackURL = null
```

Comment:
Can we please, config `slackURL` using ENV variable for this script, and update README of this script accordingly.